partition_tables/mch2022.csv

```diff
@@ -1,4 +1,8 @@
-nvs,      data, nvs,     0x9000,   16K,
-phy_init, data, phy,     0xF000,   4K,
-factory,  app,  factory, ,  5M,
-locfd,    data, fat,     , 10M,
+# 	ESP-IDF Partition Table					
+# 	Name,	Type,	SubType,	Offset,	Size,	Flags
+	nvs,	data,	nvs,	0x9000,	16K,	
+	otadata,	data,	ota,	0xD000,	8K,	
+	phy_init,	data,	phy,	0xF000,	4K,	
+	ota_0,	0,	ota_0,	0x10000,	3200K,	
```

Comment:
Because this partition table isn't actually being used. Neither is the bootloader. The actual partition table has two 1600K OTA partitions in the space of the one ota_0 partition in this fake table.